src/component/play-video/play-video.component.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
 
 const PlayVideo = ({ youtubeVideoId }) => {
-  return <h1>VIDEO: {youtubeVideoId}</h1>;
+  return (
+    <iframe
+      width="560"
```

Comment:
is this a good approach to do fixed width?
I suggest to use 100% width and a ratio of 16/9 to make it fluid and dynamic